src/viur/core/request.py

```diff
@@ -357,10 +359,9 @@ def _process(self):
                     if filename := conf["viur.mainApp"].render.getTemplateFileName((f"{error_info['status']}", "error"),
                                                                                    raise_exception=False):
                         template = conf["viur.mainApp"].render.getEnv().get_template(filename)
-                        res = template.render(error_info)
-
-                        # fixme: this might be the viur/core/template/error.html ...
-                        extendCsp({"style-src": ['sha256-Lwf7c88gJwuw6L6p6ILPSs/+Ui7zCk8VaIvp8wLhQ4A=']})
+                        nonce = utils.generateRandomString(16)
+                        res = template.render(error_info, nonce=nonce)
+                        extendCsp({"style-src": [f"nonce-{nonce}"]})
                     else:
                         res = f"""<html><h1>{error_info["status"]} - {error_info["reason"]}"""
```

Comment:
This isn't valid HTML.

Take look at https://gist.github.com/cmalven/1885287 for minimal examples. 

I want to rework this part too

Reply:
um yeah. There I forgot to pay attention to my own reminder. Thank you, I have done it